Local navigation: Add component (#1468)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/content/docs/5.3/migration.md

Reviewer: Missing new Sass and CSS variables in this file

- **Accordion**
  - <span class="badge bg-warning">Warning</span> Accordion rendering has been improved for better accessibility. Although it is transparent for basic usage, you might want to check the rendering of your custom components if they are composed of an accordion.

- **Local navigation**

>   - <span class="badge bg-success">New</span> We now provide the local navigation as a new component and its dark variant.

Reviewer: ```suggestion
  - <span class="badge bg-success">New</span> Added Local Navigation as a new component.
```
